src/inspector_agent.cc

```diff
@@ -53,7 +53,20 @@ void PrintDebuggerReadyMessage(int port) {
 }
 
 bool AcceptsConnection(inspector_socket_t* socket, const std::string& path) {
-  return 0 == path.compare(0, sizeof(DEVTOOLS_PATH) - 1, DEVTOOLS_PATH);
+  return StringEqualNoCaseN(path.c_str(), DEVTOOLS_PATH,
+                            sizeof(DEVTOOLS_PATH) - 1);
+}
+
+void Escape(std::string* string) {
+  for (std::string::iterator i = string->begin(); i < string->end(); i++) {
```

Comment:
Not exactly wrong but I think you could C++11-ify (and succinct-ify, that's definitely a word) to `for (auto& c : *string) {`?